phalcon/config.zep

```diff
@@ -223,6 +270,29 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public static function setPathDelimiter(string! delimiter = null) -> void
```

Comment:
Another point. The toArray method uses get_object_vars which would return the _delimiter object property in each array.